e2e/dractions/crud.go

```diff
@@ -46,23 +47,38 @@ func getDRPC(client client.Client, namespace, name string) (*ramen.DRPlacementCo
 	return drpc, nil
 }
 
-func createDRPC(client client.Client, drpc *ramen.DRPlacementControl) error {
+func createDRPC(ctx types.Context, client client.Client, drpc *ramen.DRPlacementControl) error {
+	log := ctx.Logger()
+
 	err := client.Create(context.Background(), drpc)
 	if err != nil {
 		if !errors.IsAlreadyExists(err) {
 			return err
 		}
-		// ctx.Log.Info("drpc " + drpc.Name + " already Exists")
+
+		log.Debugf("drpc \"%s/%s\" already exist", drpc.Namespace, drpc.Name)
 	}
 
+	// pretty-print the drpc specification
+	drpcSpecJSON, err := json.MarshalIndent(drpc.Spec, "", "  ")
```

Comment:
Since we are in the context of createDRPC(), we need shorter name. `jsonSpec` or `spec` would be fine.